parser-typechecker/src/Unison/Codebase/SqliteCodebase/MigrateSchema12.hs

```diff
@@ -104,18 +106,26 @@ import UnliftIO.Exception (bracket_, onException)
 --    * [x] Update the schema version in the database after migrating so we only migrate
 --    once.
 
+verboseOutput :: Bool
+verboseOutput =
+  isJust (unsafePerformIO (lookupEnv "UNISON_MIGRATION_DEBUG"))
+{-# NOINLINE verboseOutput #-}
```

Comment:
What would happen if we left off `NOINLINE`?

Reply:
Not much in this case, but not inlining unsafePerformIO'd cafs is standard practice anyway.